client/src/partials/home/details.html

```diff
@@ -1,23 +1,23 @@
 <div class="flex-header">
   <div class="bhima-title">
     <ol class="headercrumb">
-      <li class="static" translate>TREE.ADMIN</li>
+      <li class="static" style="text-transform : uppercase">{{ "HOME.BHIMA" | translate }}</li>
```

Comment:
You can use `class="text-uppercase"` for this to keep it bootstrapp-y.  Also, the `translate` filter is a regression.